.github/workflows/pre_release_prepare.yml

```diff
@@ -44,36 +44,36 @@ jobs:
           fi
           
           
-          if git ls-remote --heads origin release/${MAJOR_MINOR}.x | grep -q "release/${MAJOR_MINOR}.x"; then
+          if git ls-remote --heads origin release/v${MAJOR_MINOR}.x | grep -q "release/v${MAJOR_MINOR}.x"; then
             if [ "$IS_PATCH" = "true" ]; then
-              git fetch origin release/${MAJOR_MINOR}.x
-              echo "Branch release/${MAJOR_MINOR}.x already exists, checking out."
-              git checkout "release/${MAJOR_MINOR}.x"
+              git fetch origin release/v${MAJOR_MINOR}.x
+              echo "Branch release/v${MAJOR_MINOR}.x already exists, checking out."
+              git checkout "release/v${MAJOR_MINOR}.x"
             else 
-              echo "Error, release series branch release/${MAJOR_MINOR}.x exist for non-patch release"
+              echo "Error, release series branch release/v${MAJOR_MINOR}.x exist for non-patch release"
               echo "Check your input or branch"
               exit 1
             fi
           else
             if [ "$IS_PATCH" = "true" ]; then
-              echo "Error, release series branch release/${MAJOR_MINOR}.x NOT exist for patch release"
+              echo "Error, release series branch release/v${MAJOR_MINOR}.x NOT exist for patch release"
               echo "Check your input or branch"
               exit 1
             else 
-              echo "Creating branch release/${MAJOR_MINOR}.x."
-              git checkout -b "release/${MAJOR_MINOR}.x"
-              git push origin "release/${MAJOR_MINOR}.x"
+              echo "Creating branch release/v${MAJOR_MINOR}.x."
+              git checkout -b "release/v${MAJOR_MINOR}.x"
+              git push origin "release/v${MAJOR_MINOR}.x"
             fi
           fi
           
-          git checkout -b "${VERSION}_release"
-          git push origin "${VERSION}_release"
+          git checkout -b "v${VERSION}_release"
+          git push origin "v${VERSION}_release"
```

Comment:
This doesn't actually need `v`, but it's fine.

Reply:
Just making the branch names consistent